Makefile

```diff
@@ -66,6 +66,7 @@ vet: ## Vet the code and dependencies
 		if [ "${CI}" = true ]; then\
 			exit 1;\
 		fi;}
+	cd test && go vet ./...
```

Comment:
go vet ./... should include all sub directories, it's not module based afaik

Reply:
From my test, seems the root dir `go vet ./...` cannot detect e2etest folder compiling error

Reply:
what about go vet ./test/... ?

Reply:
I just tried many `go vet` commands, seems only `cd test && go vet ./...` can work